src/data/actions/grades.js

```diff
@@ -102,6 +102,7 @@ const fetchMatchingUserGrades = (courseId, searchText, cohort, track) => (
           data.next,
         ));
         dispatch(finishedFetchingGrades());
+        dispatch(updateBanner(!!showSuccess));
```

Comment:
no need for `!!` What that does is cast a truthy/falsey to a boolean. Since we already started with a boolean there is no need for it.